tools/cpp/windows_cc_configure.bzl

```diff
@@ -190,15 +190,17 @@ def _find_vcvarsall_bat_script(repository_ctx, vc_path):
   return vcvarsall
 
 
-def _find_env_vars(repository_ctx, vc_path):
+def setup_vc_env_vars(repository_ctx, vc_path):
   """Get environment variables set by VCVARSALL.BAT. Doesn't %-escape the result!"""
```

Comment:
What is %-escaping?

Reply:
Escape % for strings, it would corrupt the CROSSTOOL file because we use %{var_name} for replacement.
See https://github.com/bazelbuild/bazel/blob/db80298391bd953b1a1f8e9bb83c2d4c2f3747b4/tools/cpp/lib_cc_configure.bzl#L18